doc/tutorial/Client.lhs

```diff
@@ -27,8 +27,8 @@ Also, we need examples for some domain specific data types:
 
 ``` haskell
 data Position = Position
-  { x :: Int
-  , y :: Int
+  { xCoord :: Int
```

Comment:
Down in the file we print `Position`, and the `x` and `y` field names are in the example output